lispy.el

```diff
@@ -8563,14 +8563,19 @@ When ARG is non-nil, unquote the current string."
 
 (defvar lispy-mode-map-evilcp
   (let ((map (copy-keymap lispy-mode-map-base)))
-    (define-key map (kbd "M-)") 'lispy-close-round-and-newline)
+    ;; pairs
     (define-key map (kbd "[") 'lispy-open-square)
     (define-key map (kbd "]") 'lispy-close-square)
     (define-key map (kbd "{") 'lispy-open-curly)
     (define-key map (kbd "}") 'lispy-close-curly)
```

Comment:
I you are okay with keeping [ and ] for lispy-forward and lispy-backward, I have made two proposals below that let you do this.